qa/rpc-tests/wallet_deprecation.py

```diff
@@ -76,31 +74,39 @@ def verify_disabled(self, function):
                 "failed with '%s'" % errorString if len(errorString) > 0 else "succeeded",
             ))
 
-    def test_case(self, start_mode, features_to_allow, expected_state):
+    def test_case(self, start_mode, features_to_allow, expected_state, default_enabled, default_disabled):
         stop_nodes(self.nodes)
         wait_bitcoinds()
         start_mode(features_to_allow)
 
-        for function in DEFAULT_ENABLED:
-            expected_state(function)
-        for function in DEFAULT_DISABLED:
-            expected_state(function)
+        for function in default_enabled:
+            if function in TESTABLE_FEATURES:
+                expected_state(function)
+        for function in default_disabled:
+            if function in TESTABLE_FEATURES:
+                expected_state(function)
 
     def run_test(self):
+        dep_info = self.nodes[0].getdeprecationinfo()
+        default_enabled = dep_info['deprecated_features']
+        default_disabled = dep_info['disabled_features']
```

Comment:
We should assert that `TESTABLE_FEATURES` is contained within the set `default_enabled + default_disabled`.